internal/configs/static_evaluator.go

```diff
@@ -116,3 +117,7 @@ func (s StaticEvaluator) DecodeBlock(body hcl.Body, spec hcldec.Spec, ident Stat
 	diags = append(diags, valDiags...)
 	return val, diags
 }
+
+func (s StaticEvaluator) EvalContext(ident StaticIdentifier, refs []*addrs.Reference) (*hcl.EvalContext, tfdiags.Diagnostics) {
```

Comment:
Most functions on this struct return hcl.Diagnostics.  Perhaps we want to to the conversion here?

Reply:
Fixed in [0abfc55](https://github.com/opentofu/opentofu/pull/1728/commits/0abfc55900be650c85710f65b0c67f614976345d)